python-backend/app/api/applications/resources/application_status_code.py

```diff
@@ -0,0 +1,14 @@
+from flask_restplus import Resource, reqparse
+from flask import request
+from datetime import datetime
+
+from app.extensions import api
+from ..models.application_status_code import ApplicationStatusCode
+from app.api.utils.resources_mixins import UserMixin, ErrorMixin
+from app.api.utils.access_decorators import requires_role_mine_view
+
+
+class ApplicationStatusCodeResource(Resource, UserMixin, ErrorMixin):
+    @requires_role_mine_view
+    def get(self):
+        return [x.json() for x in ApplicationStatusCode.find_all_active_application_status_code()]
```

Comment:
`ApplicationStatusCode.find_all_active_application_status_code()` can return `None`

From what I can tell, `None.json()` will throw an exception. You should probably update the method to return an empty list instead of `None`, or handled it here with a null check

Reply:
Yes, I didn't think about the value error section. I have modified this to check and return an empty list if there is a value error.